DataStructuresMemories/ViewControllers/DataStructViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  DataStructViewController.swift
+//  DataStructuresMemories
+//
+//  Created by Oleksii Kolomiiets on 4/13/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import UIKit
+
+class DataStructViewController: UIViewController {
+    
+    @IBOutlet weak var textInsideLable: UILabel!
+    
+    var tappedCell: DataStructModel?
```

Comment:
why don't we use protocol there?

Reply:
I agree with you, it will be much better to use protocol there.